{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -0,0 +1,56 @@
+PYTHON_VERSION := 3.11
+WHEEL_VERSION := 0.43.0
+
+.DEFAULT_GOAL := venv/bin/activate
+
+venv/bin/activate: requirements-dev.txt
```

Comment:
maybe just `venv`?

Reply:
That's not how makefile works, "functions" below check for existance of venv/bin/activate file if it doesn't exist then makefile will run this "function"